### PR TITLE
Math-Magicians-Events

### PR DESCRIPTION
**Pull request by ab-noori**
## Math Magicians: Events

  ###  🌟Features:
- Components directory is created.
- [`Calculator`](https://github.com/ab-noori/math-magicians/blob/7923ce084d72aea101deb295c1708abd086cf0ed/src/components/Calculator.js) component is added.
- Calculator component is [`styled`](https://github.com/ab-noori/math-magicians/blob/7923ce084d72aea101deb295c1708abd086cf0ed/src/App.css) with css.
- [`Button`](https://github.com/ab-noori/math-magicians/blob/204705db003d09071ec3b74d710fe4f27d4b22c8/src/components/Button.js) added as a separate component.
- [`Logic`](https://github.com/ab-noori/math-magicians/tree/events/src/components/logic) files are added.
- Button [`event handler`](https://github.com/ab-noori/math-magicians/blob/204705db003d09071ec3b74d710fe4f27d4b22c8/src/components/Calculator.js#L12) added and embedded to each button.
### [ 🚀 Project Demo](https://ab-noori.github.io/math-magicians/)

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,36 +1,55 @@
...
+        <Button className="clear" text="AC" onClick={() => handleClick('AC')} />
+        <Button className="plus-minus" text="+/-" onClick={() => handleClick('+/-')} />
+        <Button className="percent" text="%" onClick={() => handleClick('%')} />
+        <Button className="operator" text="÷" onClick={() => handleClick('÷')} />
+      </div>
+      <div className="row">
+        <Button className="number" text="7" onClick={() => handleClick('7')} />
+        <Button className="number" text="8" onClick={() => handleClick('8')} />
+        <Button className="number" text="9" onClick={() => handleClick('9')} />
+        <Button className="operator" text="×" onClick={() => handleClick('×')} />
```
Comment (m05e5) [inline src/components/Calculator.js]: - [ ] In operate.js, the expected sign for multiplications is `x` but in your Calculator.js you are using `×` which leads to an error. kindly fix it
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @ab-noori , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [x] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (ab-noori) [reply src/components/Calculator.js]: Hi @m05e5, 
Thanks for reviewing my project and mentioning the error.
I have just modified the multiplication sign in [`Calculator.js`](https://github.com/ab-noori/math-magicians/blob/6399e96b5128b7d02dbe8b659f25e899d71de1cd/src/components/Calculator.js#L32) component.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @ab-noori 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**